### ISSUE TITLE
Time Complexity of Training each fields

### ISSUE THREAD

**Issue by vijayinfy0306** [naiveHobo/InvoiceNet#106]
For training it is taking a lot of time like 9-10 hours for each field only if I reduced the training steps from 50,000(default) to 1,000. Is there any way to decrease the time for training and Moreover what is the usage case of steps and what will happen if reduced the steps count (I mean is it effect the accuracy) ? 

python train.py --field round_off --batch_size 8 --data_dir <processed_data_path> --steps 50000
